sources/mysql/toddl.go

```diff
@@ -85,7 +85,7 @@ func toSpannerTypeInternal(srcType schema.Type, spType string) (ddl.Type, []inte
 		case ddl.String:
```

Comment:
Is the FLOAT32 datatype being added for PG dialect as well? If yes, what is the DDL for it and if no then that handling is missing from the code throughout.

Reply:
Yes, FLOAT32 datatype is added for PG dialect and is represented by "float4" or "real" keywords.

Reply:
In that case, it needs to be added in STANDARD_TYPE_TO_PGSQL_TYPEMAP and PGSQL_TO_STANDARD_TYPE_TYPEMAP in ast.go and test cases for it in ast_test.go

Reply:
done.